zk_toolbox/crates/zk_supervisor/src/commands/contracts.rs

```diff
@@ -0,0 +1,41 @@
+use common::{cmd::Cmd, logger, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use crate::messages::{
+    MSG_BUILDING_CONTRACTS, MSG_BUILDING_CONTRACTS_SUCCESS, MSG_BUILDING_L1_CONTRACTS_SPINNER,
+    MSG_BUILDING_L2_CONTRACTS_SPINNER, MSG_BUILDING_SYSTEM_CONTRACTS_SPINNER,
+    MSG_CONTRACTS_DEPS_SPINNER,
+};
+
+pub fn run(shell: &Shell) -> anyhow::Result<()> {
+    logger::info(MSG_BUILDING_CONTRACTS);
+
+    let ecosystem = EcosystemConfig::from_file(shell)?;
+    let link_to_code = ecosystem.link_to_code.clone();
+
+    let spinner = Spinner::new(MSG_CONTRACTS_DEPS_SPINNER);
+    let _dir_guard = shell.push_dir(&link_to_code);
+    Cmd::new(cmd!(shell, "yarn install")).run()?;
+    spinner.finish();
```

Comment:
can we add an ability to choose what contracts we want to build

Reply:
I think that optional enum, without multiple choices is better.

Reply:
3 arguments instead of vec?

Reply:
I mean, it'd be more convinient to call it 
`zks contracts l1` instead of `zks contracts -c l1` and I don't see a lot of value, in having multiple arguments, because it's only 3 options